docs/resources/azure_dns_zones_resource.md

```diff
@@ -0,0 +1,109 @@
+---
+title: About the azure_dns_zones_resource Resource
+platform: azure
+  ---
+
+# azure_dns_zones_resource
+
+Use the `azure_dns_zones_resource` InSpec audit resource to test properties of an Azure DNS zone.
+
+## Azure REST API version, endpoint, and HTTP client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the HTTP client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials,
+refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+This resource requires either the `resource_group` and DNS zone resource `name`, or the `resource_id`.
+
+```ruby
+describe azure_dns_zones_resource(resource_group: 'RESOURCE_GROUP_NAME', name: 'DNS_ZONE_NAME') do
+  it { should exist }
+end
+```
+
+or
+
+```ruby
+describe azure_dns_zones_resource(resource_id: 'DNS_ZONE_RESOURCE_ID') do
+  it { should exist }
+end
+```
+
+## Parameters
+
+| Name                           | Description                                                                      |
+|--------------------------------|----------------------------------------------------------------------------------|
+| resource_group                 | Azure resource group that the targeted resource resides in.                      |
+| name                           | Name of the DNS zone to test.                                                    |
+| resource_id                    | The resource ID the DNS zone to test.                                            |
```

Comment:
Could you specify this as optional parameter .. it is confusing if it is mandatory or not

Reply:
done